src/audio/audio.cpp

```diff
@@ -366,9 +545,38 @@ bool Audio::tryCaptureSamples(uint8_t* buf, int framesize)
 
     memset(buf, 0, framesize * 2 * av_DefaultSettings.audio_channels); // Avoid uninitialized values (Valgrind)
     alcCaptureSamples(Audio::alInDev, buf, framesize);
+
+    if (inputVolume != 1)
+    {
+        int16_t* bufReal = reinterpret_cast<int16_t*>(buf);
+        for (int i = 0; i < framesize; ++i)
+        {
+            int sample = bufReal[i] * pow(inputVolume, 2);
+
+            if (sample < std::numeric_limits<int16_t>::min())
+                sample = std::numeric_limits<int16_t>::min();
+            else if (sample > std::numeric_limits<int16_t>::max())
+                sample = std::numeric_limits<int16_t>::max();
+
+            bufReal[i] = sample;
+        }
+    }
+
     return true;
 }
 
+void Audio::pauseOutput()
+{
+    getInstance().PauseOutput();
+}
+
+void Audio::PauseOutput()
```

Comment:
@agilob Both of these `pauseOutput` methods are unused - and implementation is wrong. Should I remove?